src/main/java/teammates/ui/controller/InstructorCourseDetailsPageData.java

```diff
@@ -20,11 +24,148 @@ public InstructorCourseDetailsPageData(AccountAttributes account) {
 
     public InstructorAttributes currentInstructor;
     public CourseDetailsBundle courseDetails;
-    public List<StudentAttributes> students;
     public List<InstructorAttributes> instructors;
     public String studentListHtmlTableAsString;
+    public ElementTag giveCommentButton;
+    public ElementTag courseRemindButton;
+    public CourseDetailsStudentsTable studentsTable;
     
-    public String getInstructorCourseRemindLink() {
+    public void init(InstructorAttributes currentInstructor, CourseDetailsBundle courseDetails, 
+                     List<InstructorAttributes> instructors, List<StudentAttributes> students) {
+        this.currentInstructor = currentInstructor;
+        this.courseDetails = courseDetails;
+        this.instructors = instructors;
+        
+        boolean isDisabled = !currentInstructor.isAllowedForPrivilege(
+                                                    Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS);
+        
+        String content = "<span class=\"glyphicon glyphicon-comment glyphicon-primary\"></span>";
```

Comment:
It's icon of bootstrap http://getbootstrap.com/components/#glyphicons
In the UI, it appears right before the text of the button so it's also counted as the content of the button.